.env.sample

```diff
@@ -10,3 +10,9 @@ PGOPTIONS="-c search_path=storage"
 FILE_SIZE_LIMIT=52428800
 STORAGE_BACKEND=s3
 FILE_STORAGE_BACKEND_PATH=./data
+
+# Multitenant
+X_FORWARDED_HOST_REGEXP=
+SUPABASE_URL=
```

Comment:
reminder to use the read replica URL here since we are only reading always

Reply:
Renamed SUPABASE_URL to SUPABASE_READ_ONLY_URL for clarity.